lib/cybersource/api.rb

```diff
@@ -276,6 +279,16 @@ def should_credit?(kb_payment_id, context, options = {})
         (now - transaction.created_at) >= threshold
       end
 
+      def before_gateways(kb_transaction, last_transaction, payment_source, amount_in_cents, currency, options, context = nil)
+        # Provide necessary information for merchant descriptor
+        if options[:merchant_descriptor].present?
+          options[:merchant_descriptor][:is_amex] = (payment_source.brand == 'american_express')
```

Comment:
`.brand` may not be defined in case of tokenized card (https://github.com/killbill/killbill-plugin-framework-ruby/blob/master/lib/killbill/helpers/active_merchant/payment_plugin.rb#L495). In such cases, we should:

1. Check if the brand is passed via plugin properties (property `cc_type`, see https://github.com/killbill/killbill-plugin-framework-ruby/blob/master/lib/killbill/helpers/active_merchant/payment_plugin.rb#L177)
2. As a last resort, retrieve the payment method again to see if the brand was stored on disk